### PR TITLE
Xbox rumble function

### PR DESCRIPTION
**Pull request by Mboesn**

### CODE REVIEW

File: src/main/java/frc/robot/utils/TrigonXboxController.java [hunk 1]
```diff
@@ -91,4 +99,34 @@ public double getDeltaTriggers(Hand positiveHand) {
     public double getDeltaTriggers() {
         return getTriggerAxis(Hand.kRight) - getTriggerAxis(Hand.kLeft);
     }
-}
+
+    /** This sets both motors to 100% rumble or 0% rumble. */
+    public void setRumble(boolean isRumble) {
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonXboxController.java]: public void setRumble(boolean isRumble) {
just do:
    `public void setRumble(double power) {
            setRumble(RumbleType.kLeftRumble, power);
            setRumble(RumbleType.kRightRumble, power);
     }`

File: src/main/java/frc/robot/utils/TrigonXboxController.java [hunk 2]
```diff
@@ -91,4 +99,34 @@ public double getDeltaTriggers(Hand positiveHand) {
...
+     * @param quantity the number of times to rumble.
+     */
+    public void intermittentRumble(int quantity) {
+        rumbleAmount = quantity * 2 - 1;
+    }
+
+    public void notifierPeriodic() {
+        if (rumbleAmount > 0) {
+            if (rumbleAmount % 2 == 1)
+                setRumble(true);
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonXboxController.java]: here do: 
`setRumble(1); // Full rumble power`

File: src/main/java/frc/robot/utils/TrigonXboxController.java [hunk 3]
```diff
@@ -91,4 +99,34 @@ public double getDeltaTriggers(Hand positiveHand) {
...
+            else
+                setRumble(false);
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonXboxController.java]: here do: 
`setRumble(0);`

File: src/main/java/frc/robot/utils/TrigonXboxController.java [hunk 4]
```diff
@@ -91,4 +99,34 @@ public double getDeltaTriggers(Hand positiveHand) {
...
+     * Rumbles the controller the desired amount.
+     *
+     * @param quantity the number of times to rumble.
+     */
+    public void intermittentRumble(int quantity) {
+        rumbleAmount = quantity * 2 - 1;
+    }
+
+    public void notifierPeriodic() {
+        if (rumbleAmount > 0) {
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonXboxController.java]: if (rumbleAmount > 0) {
to 
        if (rumbleAmount >= 0) {

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
come on

File: src/main/java/frc/robot/utils/TrigonXboxController.java [hunk 5]
```diff
@@ -1,9 +1,11 @@
 package frc.robot.utils;
 
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.Notifier;
 import edu.wpi.first.wpilibj.XboxController;
 
 public class TrigonXboxController extends XboxController {
+    private static final double kIntermittentRumbleTime = 0.5;
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonXboxController.java]: **Make it 0.15** we tested it and this value works great

**Review by HilelAmiel — COMMENTED** [review]
important thing

**Comment by EliyaFishman** [comment]
[add rumble method to TrigonXboxController class](https://trello.com/c/iOp3cYSj/140-add-rumble-method-to-trigonxboxcontroller-class)

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**